index.html

```diff
@@ -68,12 +68,46 @@ <h1>I’m Patrick Ndimukulaga Glad to see you!</h1>
         </div>
       </div>
     </div>
+    <!--popup menu-->
+    <!-- <div class="modal">
+      <div class="overlay"></div>
+      <div class="modal-content">
+        <div class="modal-header">
+          <h2 class="modal-heading">Tonic</h2><img src="./images/cross-icon-popup.svg">
+        </div>
+        <div class="modal-subheading">
+          <span class="modal-canopy">CANOPY</span>
+          <span class="modal-gray-matter">• Back End Dev • 2015</span>
+        </div>
+        <img class="modal-image" src="./images/card1.svg" alt="">
+        <div class="modal-desktop-content">
+          <p class="modal-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+            Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+            typesetting industry.</p>
+          <div class="modalListWrapper">
+            <ul class="card-list modal-card-list">
+              <li class="card-list-items">html</li>
+              <li class="card-list-items">css</li>
+              <li class="card-list-items">Javascript</li>
+            </ul>
+            <hr class="modal-separator">
+            <div class="modalBtn">
+                <a class="modal-card-button" href="https://samwfelice.github.io/Portfolio/"
+                target="_blank">See Live <img src="./images/github-icon.svg"></a>
+                <a class="modal-card-button" href="https://github.com/samWfelice/Portfolio" target="_blank">See Source <img
+                  src="./images/source-icon.svg"></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div> -->
+    <!--1st porfolio-->
```

Comment:
please remove the unused commented code in order to keep your file clean and professional